src/Command/RunCommand.php

```diff
@@ -227,20 +227,14 @@ private function getCSVOption(InputInterface $input, string $option): array
             return [];
         }
 
-        if (false === is_array($result)) {
-            return array_filter(
-                explode(',', (string) $result),
-                static function ($value) {
-                    return false === empty($value);
-                }
-            );
+        if (!is_array($result)) {
+            return array_filter(explode(',', (string) $result));
         }
 
-
         return $result;
     }
 
-    protected function createFinder(InputInterface $input): PHPFinder
+    private function createFinder(InputInterface $input): PHPFinder
```

Comment:
Isn't this technically a BC break? Class `RunCommand` is not final, so someone can extend it and call `createFinder`.

Reply:
Technically yes. But practically I don't think that anybody did this :)

Reply:
reverted